### ISSUE TITLE
languages cannot be set in the profile

### ISSUE THREAD

**Issue by martinroob** [martinroob/ngx-i18nsupport#6]
Instead of giving all languages at the commandline, it should be possible to set them in the profile.

Before:
`xliffmerge en de fr it`

After:
```
xliffmergeconfig.json:
{..
  "languages": ["en", "de", "fr", "it"],
  ..
}
commandline:
xliffmerge --profile xliffmergeconfig.json
```

**Comment by martinroob** [comment]
fixed in version 0.1

**Comment by martinroob** [comment]
fixed in version 0.1
